harmony_changelog.md

```diff
@@ -5,6 +5,7 @@
 ### Fixed
 * Fixed BIWIN eMMC memory errors
 * Fixed UI freeze after changing any volume intensively
+* Fixed debug mode
```

Comment:
I'm not sure whether this changelog entry should be added - after all it's a non-released bug with no impact on users whatsoever.

Reply:
Thanks, done ;)